src/components/Map/Map.js

```diff
@@ -1,24 +1,46 @@
-import { useState, useEffect } from "react";
-import { createMap, setCurrentLocationAsCenter } from "services/MapService";
+import { useState, useEffect } from 'react';
+import { GoogleMap, useLoadScript } from '@react-google-maps/api';
 import "components/Map/Map.scss";
 
-const MAP_ID = "map";
+const defaultLocation = { lat: 19.432, lng: -99.133 };
+const defaultZoom = 12;
```

Comment:
Constants should be named in all caps
```suggestion
const DEFAULT_LOCATION = { lat: 19.432, lng: -99.133 };
const DEFAULT_ZOOM = 12;
```
And I also think creating a constants file near the Map component might be a useful place to put them in